wp-modules/api-data/api-data.php

```diff
@@ -107,6 +127,28 @@ function delete_pattern( $request ) {
 		: new WP_REST_Response( $is_success, 400 );
 }
 
+/**
+ * Updates the list of theme names that should not show version control notifications.
+ *
+ * @param WP_REST_Request $request Full data about the request.
+ * @return WP_REST_Response
+ */
+function update_dismissed_themes( $request ) {
+	$user_id    = get_current_user_id();
+	$meta_key   = 'patternmanager_version_control_notice_dismissed_themes';
```

Comment:
Given that `patternmanager_version_control_notice_dismissed_themes` is also in `wp-modules/api-data/api-data.php`, maybe `wp-modules/api-data/api-data.php` could have a function like:

```php
function get_version_control_meta_key() {
	return 'patternmanager_version_control_notice_dismissed_themes';
}
```

Reply:
Oh yes, I missed that! Added in [37afb1a](https://github.com/studiopress/pattern-manager/pull/145/commits/37afb1a1d1a8d1d7419462840ca3fdb263ef7b38).